Procfile

```
@@ -1 +1 @@
-release: bin/rails db:migrate
+git add Procfilerelease: bin/rails db:migrate
```

gitのコマンドが入ってしまっていますね。

```suggestion
release: bin/rails db:migrate
```